config.go

```diff
@@ -56,6 +57,8 @@ type config struct {
 	Profile string `long:"profile" description:"Enable HTTP profiling on given port -- NOTE port must be between 1024 and 65536"`
 
 	PeerPort int    `long:"peerport" description:"The port to listen on for incoming p2p connections"`
+
+	// TODO(andrew.shvv) Add RPCHost?
```

Comment:
Sorry, I think I should be more verbose with this `TODO`:

That is true - we have `RPCHost`, `RPCPort` variables, but such naming introduce misconception - `RPCPort` is used for `lnd` initialisation, and for now we haven't way for changing the `lnd` `RPC` host.
```
// Next, Start the grpc server listening for HTTP/2 connections.
grpcEndpoint := fmt.Sprintf("localhost:%d", loadedConfig.RPCPort)
```

and `RPCHost` is used for `btcd` intialiasation:
```
btcdHost := fmt.Sprintf("%v:%v", cfg.RPCHost, activeNetParams.rpcPort)
```

So I think it would be better at least rename parameters.